addons/mail/static/src/discuss/call/common/call_settings.js

```diff
@@ -5,10 +5,15 @@ import { browser } from "@web/core/browser/browser";
 import { debounce } from "@web/core/utils/timing";
 import { isMobileOS } from "@web/core/browser/feature_detection";
 import { useService } from "@web/core/utils/hooks";
+import { ActionPanel } from "@mail/discuss/core/common/action_panel";
 
 export class CallSettings extends Component {
     static template = "discuss.CallSettings";
-    static props = ["*"];
+    static props = ["withActionPanel?", "*"];
+    static defaultProps = {
+        withActionPanel: true,
+    };
+    static components = { ActionPanel };
```

Comment:
Ok for now, `ActionPanel` is managed by each thread action component.

At some point we should improve the code so that thread actions that become a panel (currently all that have `{ toggle: true }`) should be displayed in an `<ActionPanel>`. Title is named after thread action, and resizable could be an option on thread action too.